Headers/AppLovinSDK.framework/Headers/ALSdkSettings.h

```diff
@@ -0,0 +1,112 @@
+ */
+@property (nonatomic, copy, nullable) NSURL *privacyPolicyURL;
+
+/**
+ * URL for your company’s terms of service. This is optional; you can enable the consent flow with or without it.
+ *
+ * This defaults to the value that you entered into your `Info.plist` file via `AppLovinConsentFlowInfo`⇒`AppLovinConsentFlowTermsOfService`.
+ *
+ * @see <a href="https://dash.applovin.com/documentation/mediation/ios/getting-started/consent-flow#info.plist">MAX Integration Guide ⇒ iOS ⇒ Consent Flow ⇒ Info.plist</a>
+ */
+@property (nonatomic, copy, nullable) NSURL *termsOfServiceURL;
+
+- (instancetype)init NS_UNAVAILABLE;
++ (instancetype)new NS_UNAVAILABLE;
+
+@end
+
+/**
+ * This class contains settings for the AppLovin SDK.
+ */
+@interface ALSdkSettings : NSObject
+
+/**
+ * Settings relating to the AppLovin consent flow.
+ */
+@property (nonatomic, strong, readonly) ALConsentFlowSettings *consentFlowSettings;
+
+/**
+ * A toggle for verbose logging for the SDK. This is set to `NO` by default. Set it to `NO` if you want the SDK to be silent (this is recommended for App Store
+ * submissions).
+ *
+ * If set to `YES` AppLovin messages will appear in the standard application log which is accessible via the console. All AppLovin log messages are prefixed
+ * with the `/AppLovinSdk: [AppLovinSdk]` tag.
+ *
+ * Verbose logging is <em>disabled</em> (`NO`) by default.
+ *
+ * @see <a href="https://dash.applovin.com/documentation/mediation/ios/getting-started/advanced-settings#enable-verbose-logging">MAX Integration Guide ⇒ iOS ⇒ Advanced Settings ⇒ Enable Verbose Logging</a>
+ */
+@property (nonatomic, assign) BOOL isVerboseLogging;
+
+/**
+ * Whether to begin video ads in a muted state or not. Defaults to `NO` unless you change this in the dashboard.
+ *
+ * @see <a href="https://dash.applovin.com/documentation/mediation/ios/getting-started/advanced-settings#mute-audio">MAX Integration Guide ⇒ iOS ⇒ Advanced Settings ⇒ Mute Audio</a>
+ */
+ // [PLP] {@link} to the part of the dashboard where you can change this?
+@property (nonatomic, assign) BOOL muted;
+
+/**
+ * Whether the creative debugger will be displayed on fullscreen ads after flipping the device screen down twice. Defaults to `YES`.
+ *
+ * @see <a href="https://dash.applovin.com/documentation/mediation/ios/testing-networks/creative-debugger">MAX Integration Guide ⇒ iOS ⇒ Testing Networks ⇒ Creative Debugger</a>
+ */
+@property (nonatomic, assign) BOOL creativeDebuggerEnabled;
+
+/**
+ * Enable devices to receive test ads by passing in the advertising identifier (IDFA) of each test device.
+ * Refer to AppLovin logs for the IDFA of your current device.
+ */
+@property (nonatomic, copy) NSArray<NSString *> *testDeviceAdvertisingIdentifiers;
+
+/**
+ * The MAX ad unit IDs that will be used for this instance of the SDK. Third-party SDKs will be initialized with the credentials configured for these ad unit
+ * IDs.
+ */
+ // [PLP] "will be used" by whom? "will be initialized" by whom?
```

Comment:
So pubs can set a specific set of Ad Unit IDs that they are going to use in their integration.